uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java

```diff
@@ -40,7 +40,7 @@
     // 2) Draw Card
     // 3) Play Card
     @PutMapping(value = "rooms/{uid}/players/{pid}")
-    public ResponseEntity<player> handlePlayerPuts(@RequestBody String json, @PathVariable String uid, @PathVariable String pid) {
+    public ResponseEntity<Object> handlePlayerPuts(@RequestBody String json, @PathVariable String uid, @PathVariable String pid) {
```

Comment:
Can we have a comment here or something to explain the differences in the objects being returned?

Reply:
Added comments. Yes, that can be a separate ticket after everything is fully tested and working. The issue is that play and draw doesn't convert to any object and having separate PUT functions need some more refactoring.